airbyte-webapp/src/test-utils/setup-tests.ts

```diff
@@ -0,0 +1,22 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import "../globals";
+
+// jsdom doesn't mock `matchMedia`, which is required by react-slick
+Object.defineProperty(window, "matchMedia", {
```

Comment:
:information_source: Rewrote this mock slightly from what we had beforehand here, to align with what `jest` actually has in their documentation: https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom